### ISSUE TITLE
TFLite file larger than expected

### ISSUE THREAD

**Issue by andynader** [larq/larq#697]
### Discussed in https://github.com/larq/larq/discussions/696

<div type='discussions-op-text'>

<sup>Originally posted by **andynader** September  7, 2021</sup>
Hi, When I train the following architecture: 

    kwargs = {
        "input_quantizer": "ste_sign",
        "kernel_quantizer": "ste_sign",
        "kernel_constraint": "weight_clip",
    }

    binary_net = tf.keras.Sequential(
        [
            tf.keras.layers.Input(shape=(x_train.shape[1], x_train.shape[2])),
            lq.layers.QuantConv1D(32, (3,), padding="same", use_bias=False, **kwargs),
            lq.layers.QuantConv1D(32, (3,), padding="same",  use_bias=False, **kwargs),
            tf.keras.layers.MaxPooling1D(pool_size=2),
            lq.layers.QuantConv1D(64, (3,), padding="same",  use_bias=False, **kwargs),
            lq.layers.QuantConv1D(64, (3,), padding="same", use_bias=False, **kwargs),
            tf.keras.layers.MaxPooling1D(pool_size=4),
            lq.layers.QuantConv1D(128, (3,), padding="same",  use_bias=False, **kwargs),
            lq.layers.QuantConv1D(128, (3,), padding="same",  use_bias=False, **kwargs),
            tf.keras.layers.MaxPooling1D(pool_size=6),
            lq.layers.QuantConv1D(256, (3,), padding="same",  use_bias=False, **kwargs),
            lq.layers.QuantConv1D(256, (3,), padding="same", use_bias=False, **kwargs),
            tf.keras.layers.MaxPooling1D(pool_size=6),
            lq.layers.QuantConv1D(256, (3,), padding="same",  use_bias=False, **kwargs),
            lq.layers.QuantConv1D(256, (3,), padding="same", use_bias=False, **kwargs),
            tf.keras.layers.UpSampling1D(size=6),
            lq.layers.QuantConv1D(128, (3,), padding="same",  use_bias=False, **kwargs),
            lq.layers.QuantConv1D(128, (3,), padding="same",  use_bias=False, **kwargs),
            tf.keras.layers.UpSampling1D(size=6),
            lq.layers.QuantConv1D(64, (3,), padding="same",  use_bias=False, **kwargs),
            lq.layers.QuantConv1D(64, (3,), padding="same",  use_bias=False, **kwargs),
            tf.keras.layers.UpSampling1D(size=4),
            lq.layers.QuantConv1D(32, (3,), padding="same", use_bias=False, **kwargs),
            lq.layers.QuantConv1D(32, (3,), padding="same", use_bias=False, **kwargs),
            tf.keras.layers.UpSampling1D(size=2),
            lq.layers.QuantConv1D(3, (3,), padding="same",  use_bias=False, **kwargs),
            lq.layers.QuantConv1D(3, (3,), padding="same",  use_bias=False, **kwargs),
            tf.keras.layers.Flatten(),
            lq.layers.QuantDense(288, use_bias=False, **kwargs)
        ]
    )

lq.models.summary() suggests that the model should be around 150KB, yet when I convert it to a TFLite file using lce.convert_keras_model(), the file ends up being 5MB, which is the size of the floating point equivalent of the network. In addition, when I benchmark this model on a Pi 4, it ends up being equivalent in inference time to a full precision network. Any idea why this is happening?</div>

**Comment by andynader** [comment]
The problem was that Larq TFLite doesn't currently support 1D  binary convolutions, as @Tombana explained in the discussion thread. Switching to 2D convolutions fixed this issue.
